test/e2e/csv_e2e_test.go

```diff
@@ -1551,7 +1553,7 @@ var _ = Describe("ClusterServiceVersion", func() {
 		// Fetch cluster service version again to check for unnecessary control loops
 		sameCSV, err := fetchCSV(crc, csv.Name, testNamespace, csvSucceededChecker)
 		Expect(err).ShouldNot(HaveOccurred())
-		compareResources(GinkgoT(), fetchedCSV, sameCSV)
+		Expect(equality.Semantic.DeepEqual(fetchedCSV, sameCSV)).Should(BeTrue(), "CSV is same: "  + diff.ObjectDiff(fetchedCSV, sameCSV))
```

Comment:
I think I'm largely fine with deflating the compareResources logic at the individual call sites but I can't wrap my head around what that `"CSV is same: "  + diff.ObjectDiff(fetchedCSV, sameCSV)` would look like when the assertion has failed. Do we need that prefixed "CSV is same: ..." or can we just use `diff.ObjectDiff(...)` to provide debuggability when the assertion has failed?

Reply:
It looks like this is also failing the sanity check too.

Reply:
OK.  I change to just the diff there.  Thanks!